scenarios/bigquery.py

```diff
@@ -17,79 +17,78 @@
 """Runs a bigquery query, filters the results, and uploads the filtered and complete data to GCS."""
 
 import argparse
```

Comment:
Plesae sort, note that yaml (a 3p library) should be in a separate section:

https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting